tf/main.tf

```
@@ -121,3 +121,22 @@ resource "cloudflare_access_policy" "policy" {
     email = ["CHANGEME@example.com"]
   }
 }
+
+resource "cloudflare_ruleset" "wildebeest_config_rules" {
+  zone_id     = var.cloudflare_zone_id
+  name        = "Config rules ruleset"
+  kind        = "zone"
+  phase       = "http_config_settings"
+
+  rules {
+    action = "set_config"
+    action_parameters {
+      "autominify": {
+        "js": false
+      }
```

just need to tidy up the syntax on this one

```suggestion
      autominify {
        js = false
      }
```